anyway/widgets/widget_utils.py

```diff
@@ -123,3 +125,30 @@ def format_2_level_items(
             series_data.append({BE_CONST.LKEY: l2, BE_CONST.VAL: num})
         res.append({BE_CONST.LKEY: l1, BE_CONST.SERIES: series_data})
     return res
+
+
+def second_level_fill_and_sort(data: dict, default_order: dict) -> dict:
+    for num, value in data.items():
+        new_value = copy.deepcopy(default_order)

```

Comment:
in this method you are using deepcopy, but in fill_and_sort_by_numeric_range you don't. why is that?

Reply:
You are right, will be fixed